_sass/_partners.scss

```diff
@@ -0,0 +1,91 @@
+@media screen and (min-width: 280px) {
+    .each-partner {
+        width: 100%;
+        background-color: #273647;
```

Comment:
Also, if we're going to use the boxes, please use `border-radius` on them for consistency with other image/box elements we use on the site.